src/main/kotlin/gdx/liftoff/data/platforms/iOSMOE.kt

```diff
@@ -0,0 +1,201 @@
+
+        arrayOf(
+            "Default-1024w-1366h@2x~ipad.png",
+            "Default@2x~ipad.png",
+            "Default@2x.png",
+            "Default-375w-667h@2x.png",
+            "Default-375w-812h@3x.png",
+            "Default-414w-736h@3x.png",
+            "Default-568h@2x.png",
+            "Default~ipad.png",
+            "Default.png",
+            "main.cpp"
+        ).forEach {
+                project.files.add(
+                    CopiedFile(
+                        projectName = iOSMOE.ID,
+                        path = path("xcode", "ios-moe", it),
+                        original = path("generator", "ios-moe", "xcode", "ios-moe", it)
+                    )
+                )
+            }
+
+        project.files.add(
+            ReplacedContentFile(
+                projectName = iOSMOE.ID,
+                path =  path("xcode", "ios-moe", "Info.plist"),
+                original = path("generator", "ios-moe", "xcode", "ios-moe", "Info.plist"),
+                replaceMap = mapOf(Pair("%PACKAGE%", project.basic.rootPackage))
+            )
+        )
+
+        project.files.add(
+            ReplacedContentFile(
+                projectName = iOSMOE.ID,
+                path = path("xcode", "ios-moe.xcodeproj", "project.pbxproj"),
+                original = path("generator", "ios-moe", "xcode", "ios-moe.xcodeproj", "project.pbxproj"),
+                replaceMap = mapOf(
+                    Pair("%PACKAGE%", project.basic.rootPackage),
+                    Pair("%ASSET_PATH%", "assets/"),
+                    Pair("%APP_NAME%", project.basic.name)
+                )
+            )
+        )
+
+        arrayOf("Info.plist", "main.cpp").forEach {
+            project.files.add(
+                CopiedFile(
+                    projectName = iOSMOE.ID,
+                    path = path("xcode", "ios-moe-Test", it),
+                    original = path("generator", "ios-moe", "xcode", "ios-moe-Test", it)
+                )
+            )
+        }
+    }
+}
+
+class ReplacedContentFile(projectName: String = "", path: String, original: String, fileType: Files.FileType = Files.FileType.Internal, private val replaceMap: Map<String, String>) : CopiedFile(projectName, path, original, fileType) {
+    override fun save(destination: FileHandle) {
+        var input = Gdx.files.getFileHandle(original, fileType).readString("UTF-8")
+        replaceMap.forEach { (t, u) -> input = input.replace(t, u) }
+        destination.child(path).writeString(input, false, "UTF-8")
+    }
+}
+
+class iOSMOEGradleFile(val project: Project) : GradleFile(iOSMOE.ID) {
+    init {
+        dependencies.add("project(':${Core.ID}')")
+        addDependency("io.github.berstanio:gdx-backend-moe:\$gdxVersion")
+        addSpecialDependency("natives \"com.badlogicgames.gdx:gdx-platform:\$gdxVersion:natives-ios\"")
+    }
+
+    override fun getContent() = """apply plugin: 'moe'
+
+// Exclude all files from Gradle's test runner
+test { exclude '**' }
```

Comment:
I'm a little uncertain about this, but if I am reading this right, this only disables tests in the MOE module? (And they can still be run in other ways, I guess).

Reply:
Yes, exactly!